src/index.js

```diff
@@ -1,14 +1,125 @@
-import _ from 'lodash';
 import './style.css';
+import Task from './class.js';
 
-function component() {
-  const element = document.createElement('div');
+const form = document.querySelector('.todo-add');
+const input = document.querySelector('.form-field');
+const ul = document.getElementById('todo-list');
+let li;
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'ahmed'], ' ');
-  element.classList.add('hello');
+let tasks = [];
 
-  return element;
+function getTasks() {
+  if (localStorage.getItem('tasks') == null) {
+    tasks = [];
+  } else {
+    tasks = Array.from(JSON.parse(localStorage.getItem('tasks')));
+  }
+  return tasks;
 }
 
-document.body.appendChild(component());
+function prepareEdit(task, btn) {
+  btn.addEventListener('click', () => {
+    const el = document.querySelector(`[data-index="${task.index}"]`);
+    const p = el.children[1];
+    const input = document.createElement('input');
+    input.type = 'text';
+    input.value = p.textContent;
+    input.id = task.index;
+    el.insertBefore(input, p);
+    el.removeChild(p);
+    input.focus();
+    input.select();
+    input.classList.add('edit');
+  });
+}
+
+function loadTasks() {
+  const tasks = Array.from(JSON.parse(localStorage.getItem('tasks')));
+  tasks.forEach((task) => {
+    li = document.createElement('li');
+    li.classList.add('task-li');
+    li.innerHTML = `
+        <div class="task" data-index="${task.index}">
+          <input type="checkbox">
+          <p>${task.description}</p>
+          <div class="btns">
+            <button type="button" class="close-button scroll">+</button>
+            <button type="button" class="fas fa-ellipsis-v scroll" id="edit-${task.index}"></button>
+          </div>
+        </div>
+    `;
+    ul.appendChild(li);
+    const btn = document.getElementById(`edit-${task.index}`);
+    prepareEdit(task, btn);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    // getTasks();
+  });
+}
+
+function addTask() {
+  ul.innerHTML = '';
+  tasks.forEach((task) => {
+    li = document.createElement('li');
+    li.classList.add('task-li');
+    li.innerHTML = `
+        <div class="task" data-index="${task.index}">
+          <input type="checkbox">
+          <p>${task.description}</p>
+          <div class="btns">
+            <button type="button" class="close-button scroll">+</button>
+            <button type="button" class="fas fa-ellipsis-v scroll" id="edit-${task.index}"></button>
+          </div>
+        </div>
+    `;
+    ul.appendChild(li);
+    const btn = document.getElementById(`edit-${task.index}`);
+    prepareEdit(task, btn);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    // getTasks();
+    console.log(tasks);
+  });
```

Comment:
- [ ] Kindly remove all codes that are commented out. Removing such codes makes your code look clean.
- [ ] Kindly remove the `console.log` function in your code. It is a best practice to remove all `console.log` statements from your code so that you do not reveal sensitive information to visitors of your site.

Reply:
👍🏽 This has been implemented also